src/prometheus_alert_rules/pressure.rules

```diff
@@ -0,0 +1,33 @@
+groups:
+- name: HostPressure
+  rules:
+
+  # Alert for host cpu pressure - high instantanoues cpu waiting time
+  - alert: HostHighCpuWaitingTime
+    expr: irate(node_pressure_cpu_waiting_seconds_total[5m]) > 0.9
+    for: 5m
+    labels:
+      severity: info
```

Comment:
Why info? What does this imply for you specifically? I would suggest going with warning.

Reply:
The idea is to provide user some kind alert (anything other than critical) about the `cpu` and `mem` usage, and the user should spend some time to investigate what is happening when the alert is fired. So yes, I agree `warning` should be more appropriate to prompt the user to do something.